<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>861</width>
      <height>331</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="cam1_ok">
           <property name="text">
            <string>cam1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cam2_ok">
           <property name="text">
            <string>cam2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cam3_ok">
           <property name="text">
            <string>cam3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cam4_ok">
           <property name="text">
            <string>cam4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cam5_ok">
           <property name="text">
            <string>cam5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cam6_ok">
           <property name="text">
            <string>cam6</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cam1_4">
           <property name="text">
            <string>cam1_4</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="connect_pushButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>360</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>连接串口</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_ports">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>360</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1091</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
